src/dbus_api/pool.rs

```diff
@@ -52,10 +50,7 @@ fn create_filesystems(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
     let mut engine = dbus_context.engine.borrow_mut();
     let pool = get_mut_pool!(engine; pool_uuid; default_return; return_message);
 
-    let result =
-        pool.create_filesystems(&filesystems
-                                     .map(|x| (x, None))
-                                     .collect::<Vec<(&str, Option<Sectors>)>>());
+    let result = pool.create_filesystems(&filesystems.collect::<Vec<_>>());
```

Comment:
I guess it's too difficult to just revert the commits that introduced these changes?

Reply:
That was months ago. I'd be amazed if this is a straight reversion of that commit. I'll check, though!